config/plugins/visualizations/charts/static/repository/utilities/jobs.js

```diff
@@ -43,7 +43,7 @@ define( [ 'utilities/utils' ], function( Utils ) {
                     }
                     chart.state( 'failed', 'This visualization requires the \'' + parameters.tool_id + '\' tool. Please make sure it is installed. ' + message );
                     error && error();
-                          
```

Comment:
I think this is fine, committers often clean up whitespace and typos as a part of an unrelated PR.